pkgs/tools/backup/autorestic/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, fetchFromGitHub, installShellFiles, buildGoModule }:
+
+buildGoModule rec {
+  name = "autorestic";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "cupcakearmy";
+    repo = "${name}";
```

Comment:
```suggestion
    repo = name;
```

Reply:
Do you mean `repo = restic` ?

Reply:
`pname` is the same as `"${pname}"` here since it is already a string